blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/HttpMetricDataQueryServer.java

```diff
@@ -61,15 +61,21 @@ public void startServer() {
         RouteMatcher router = new RouteMatcher();
         router.get("/v1.0", new DefaultHandler());
         router.get("/v1.0/:tenantId/experimental/views/metric_data/:metricName", new HttpRollupsQueryHandler());
+
         router.post("/v1.0/:tenantId/experimental/views/metric_data", new HttpMultiRollupsQueryHandler());
+        router.post("/v2.0/:tenantId/views", new HttpMultiRollupsQueryHandler());
 
         router.get("/v2.0", new DefaultHandler());
         router.get("/v2.0/:tenantId/views/:metricName", new HttpRollupsQueryHandler());
-        router.post("/v2.0/:tenantId/views", new HttpMultiRollupsQueryHandler());
         router.get("/v2.0/:tenantId/metrics/search", new HttpMetricsIndexHandler());
         router.get("/v2.0/:tenantId/metric_name/search", new HttpMetricTokensHandler());
         router.get("/v2.0/:tenantId/events/getEvents", new HttpEventsQueryHandler(getEventsIO()));
-        router.options("/v2.0/:tenantId/events/getEvents", new HttpEventsQueryHandler(getEventsIO()));
+
+        router.options("/v2.0/:tenantId/views/:metricName", new HttpOptionsHandler());
+        router.options("/v2.0/:tenantId/views", new HttpOptionsHandler());
+        router.options("/v2.0/:tenantId/metrics/search", new HttpOptionsHandler());
+        router.options("/v2.0/:tenantId/metric_name/search", new HttpOptionsHandler());
+        router.options("/v2.0/:tenantId/events/getEvents", new HttpOptionsHandler());
 
```

Comment:
I think I prefer specific paths, although if there's a strong benefit to using regex, I'm open to it.